cypress/integration/projectSettings/project_settings.ts

```diff
@@ -402,9 +390,22 @@ describe("Project Settings when not defaulting to repo", () => {
     it("Should show three populated fields when navigating back from another page", () => {
       cy.dataCy("navitem-access").click();
       cy.dataCy("navitem-variables").click();
-      cy.dataCy("var-name-input").eq(0).should("have.value", "admin_var");
-      cy.dataCy("var-name-input").eq(1).should("have.value", "sample_name");
-      cy.dataCy("var-name-input").eq(2).should("have.value", "sample_name_2");
+      cy.dataCy("var-name-input").should("be.visible");
+      cy.dataCy("var-name-input").should(($inputs) => {
+        expect($inputs).to.have.length(3);
+        // Asserts input copy exists without asserting input order
+        // to avoid test flake if the input order rearranges.
+        const inputVals = new Set([
+          $inputs.eq(0).val(),
+          $inputs.eq(1).val(),
+          $inputs.eq(2).val(),
+        ]);
+        const copy = new Set(["admin_var", "sample_name", "sample_name_2"]);
+        expect(inputVals.size).to.equal(copy.size);
+        expect(
+          Array.from(inputVals).every((value) => copy.has(value))
+        ).to.equal(true);
+      });
```

Comment:
Final nit are we sure these weren't failing as a result of the laggy cypress tests. afaict these should always be sorted.

Reply:
I reverted the test and it passed a few times. If it fails because of sort order we will be able to see it in the video.